src/IIIFPresentation/Models/API/Collection/FlatCollection.cs

```diff
@@ -1,48 +1,45 @@
-﻿using IIIF.Presentation.V3.Strings;
+﻿using IIIF.Presentation.V3;
+using IIIF.Presentation.V3.Strings;
 using Newtonsoft.Json;
 
 namespace Models.API.Collection;
 
-public class FlatCollection
+public class FlatCollection : IIIF.Presentation.V3.Collection
 {
     [JsonProperty("@context")]
-    public List<string>? Context { get; set; }
-    
-    public string? Id { get; set; }
-    
-    public string? PublicId { get; set; }
-    
-    public PresentationType Type { get; set; }
+    public new List<string>? Context
+    {
+        get => base.Context as List<string>;
+        set => base.Context = value;
+    }
+
+    #region Overrides of Collection
 
-    public List<string> Behavior { get; set; } = new ();
+    public override string Type { get; } = nameof(FlatCollection);
 
-    public LanguageMap? Label { get; set; }
+    #endregion
+
+    public string? PublicId { get; set; }
+
+    public PresentationType PresentationType { get; set; }
 
     public required string Slug { get; set; }
-    
+
     public string? Parent { get; set; }
-    
+
     public int? ItemsOrder { get; set; }
-    
-    public List<Item>? Items { get; set; }
-    
-    public List<PartOf>? PartOf { get; set; }
-    
+
     public int TotalItems { get; set; }
 
     public View? View { get; set; }
```

Comment:
I think View might be using `JsonPropertyName` instead of `JsonProperty` for some attributes (my mistake when I built this class)

Additionally I think the `Type` in this class may need to be changed away from `PresentationType`